conans/requirements_dev.txt

```diff
@@ -4,3 +4,5 @@ parameterized>=0.6.3
 mock>=1.3.0, <1.4.0
 WebTest>=2.0.18, <2.1.0
 bottle
+PyJWT
```

Comment:
Good, we used to install ``requirements_server.txt`` for this, but I agree that this shouldn't be necessary and installing the ``dev`` requirements should be enough to run the tests too.